src/internal/connector/exchange/api/events.go

```diff
@@ -15,68 +15,83 @@ import (
 
 // CreateCalendar makes an event Calendar with the name in the user's M365 exchange account
 // Reference: https://docs.microsoft.com/en-us/graph/api/user-post-calendars?view=graph-rest-1.0&tabs=go
-func CreateCalendar(ctx context.Context, gs graph.Servicer, user, calendarName string) (models.Calendarable, error) {
+func (c Client) CreateCalendar(
+	ctx context.Context,
+	user, calendarName string,
+) (models.Calendarable, error) {
 	requestbody := models.NewCalendar()
 	requestbody.SetName(&calendarName)
 
-	return gs.Client().UsersById(user).Calendars().Post(ctx, requestbody, nil)
+	return c.stable.Client().UsersById(user).Calendars().Post(ctx, requestbody, nil)
 }
 
 // DeleteCalendar removes calendar from user's M365 account
 // Reference: https://docs.microsoft.com/en-us/graph/api/calendar-delete?view=graph-rest-1.0&tabs=go
-func DeleteCalendar(ctx context.Context, gs graph.Servicer, user, calendarID string) error {
-	return gs.Client().UsersById(user).CalendarsById(calendarID).Delete(ctx, nil)
+func (c Client) DeleteCalendar(
+	ctx context.Context,
+	user, calendarID string,
+) error {
+	return c.stable.Client().UsersById(user).CalendarsById(calendarID).Delete(ctx, nil)
 }
 
 // RetrieveEventDataForUser is a GraphRetrievalFunc that returns event data.
 // Calendarable and attachment fields are omitted due to size
```

Comment:
Not sure this is necessary:
All fields that appear under `Relationships` are not included by default. 
Both Calendar and attachments are within this category. https://learn.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0